src/pages/events/2020/03/sugcon.md

```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
+sup: The Sitecore User Group Conference (SUGCON) brings the Sitecore user community together and provides the perfect platform to gain knowledge and get inspired by the potential within Sitecore Experience Cloud, Experience Commerce, Content Hub and more. After many very successful events, we are expecting over 500 attendees to attend the event..
+date: 2020-03-26
+dateConfirmed: false
+externalLink: https://europe.sugcon.events/
+venue:
+  name: Budapest Congress Center
+  address: Budapest, Jagelló út 1-3, 1123, Hungary
+  position: 47.4897869, 19.017191507595626
+  details: 
+agenda:

```

Comment:
Can be removed?

Reply:
On event listing page the location shows - will remove details and position but I think it's useful to keep address and venue?

Reply:
I have to get used to Github but this was specifically about line 14, Agenda.